jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -970,7 +972,7 @@ pipeline {
             node(AGENT_LINUX_X64) {
                 script {
                     if (params.RC_NUMBER.toInteger() > 0) {
-                        triggerDistributionValidationWorkflow(env.version)
+                        triggerDistributionValidationWorkflow(env.revision)
```

Comment:
Since we are passing the version and qualifier together in the VERSION parameter of validation workflow. Python code changes can be ignored for now right @peterzhuamazon

Reply:
Python workflow just compose url of build manifest right?

Reply:
No it generates the [artifact url](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.19.0/latest/windows/x64/zip/dist/opensearch/opensearch-2.19.0-windows-x64.zip) directly for staging and prod , with version as main input.